src/components/Widgets/WidgetView.style.ts

```diff
@@ -0,0 +1,107 @@
+import type { BoxProps } from '@mui/material';
+import { Box, Skeleton, Typography } from '@mui/material';
+import { alpha, styled } from '@mui/material/styles';
+
+interface WidgetSkeletonContainerProps extends BoxProps {
+  welcomeScreenClosed: boolean;
+}
+
+export const WidgetSkeletonContainer = styled(
+  Box,
+)<WidgetSkeletonContainerProps>(({ theme, welcomeScreenClosed }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  backgroundColor: theme.palette.surface1.main,
+  boxShadow: '0px 8px 32px rgba(0, 0, 0, 0.08)',
+  borderRadius: theme.shape.borderRadius,
+
+  marginTop: !welcomeScreenClosed ? '24px' : theme.spacing(3.5),
+  cursor: !welcomeScreenClosed ? 'pointer' : 'auto',
+  [`@media screen and (min-height: 700px)`]: {
+    marginTop: !welcomeScreenClosed
+      ? 'calc( 50vh - 680px / 2.75 - 40px)'
+      : theme.spacing(3.5), // (mid viewheight - half-two/thirds widget height - navbar height )
+  },
```

Comment:
I guess we can avoid these magic numbers by using WidgetSkeleton the same way we use the main Widget component so everything related to its positioning is kind of inherited 🤔

Reply:
Re-used the WidgetContainer component and only have overwritten styles which cannot be re-used